usage_examples/span_srl_openai/README.md

```diff
@@ -0,0 +1,32 @@
+# Semantic Role Labeling (SRL) using LLMs
+
+This example shows how you can use a model from OpenAI for SRL in
+zero- and few-shot settings.
+
+
+We leverage the OpenAI API to detect the predicates and argument roles in a sentence.
+In the example below, we focus on the predicate "bought" and ARG-0, ARG-1, and ARG-M-LOC.
+
+First, create a new API key from
+[openai.com](https://platform.openai.com/account/api-keys) or fetch an existing
+one. Record the secret key and make sure this is available as an environmental
+variable:
+
+```sh
+export OPENAI_API_KEY="sk-..."
```

Comment:
Can you expand this to describe how the expected output looks like?

Reply:
done!